src/app/routes/applications/governance-view/governance-details/governance-details.component.ts

```diff
@@ -5,9 +5,9 @@ import { NamespaceType, PreconditionType } from 'iam-client-lib';
 import { ListType } from '../../../../shared/constants/shared-constants';
 import { IamService } from '../../../../shared/services/iam.service';
 import { LoadingService } from '../../../../shared/services/loading.service';
-import { RoleType } from '../../new-role/new-role.component';
 import { GovernanceViewComponent } from '../governance-view.component';
 import { IssuerType } from '../../new-role/models/issuer-type.enum';
+import { IFieldDefinition } from '@energyweb/iam-contracts/dist/src/types/DomainDefinitions';
```

Comment:
Isn't it directly export from `@energyweb/iam-contracts`?

Reply:
It's not reexported :/ So i need to have import like this. interface in general is exported, but in index.ts is not imported and later on reexported. The best would be just export everything imo.
![image](https://user-images.githubusercontent.com/83639710/154763426-6c46b876-2515-49fa-bea0-9eb7c23e65a5.png)

Reply:
https://github.com/energywebfoundation/iam-contracts/pull/111